middleware/ErrorHandler.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (err, req, res, next) => {
+    if (res.headersSent) {
+        res.json(err);
+    } else if (err.name === "SequelizeValidationError") {
+        res.status(400).json(err);
+    } else if (err.name === "NotFound") {
+        res.status(404).json(err);
+    } else {
```

Comment:
jangan lupa untuk handle SequelizeDatabaseError yang constraint NOT NULL (code nya bisa dicari disini https://www.postgresql.org/docs/8.2/errcodes-appendix.html) dan juga SequelizeUniqueConstraintError (untuk email nntinya)